Services/Common/OJS.Workers/OJS.Workers.ExecutionStrategies/Java/JavaProjectTestsExecutionStrategy.cs

```diff
@@ -415,6 +426,11 @@ protected virtual void ExtractUserClassNames(string submissionFilePath)
                 if (!isTestSuccessful)
                 {
                     var errorLine = output.ReadLine();
+                    if (errorLine!.StartsWith("Invalid number of test cases"))
```

Comment:
Extract this into a private constant for the file, as we use it on separate places and it has to be the same. It will improve readability and it will be safer for modifications.

Reply:
Done.